apps/website/src/components/notifications/Schedule.tsx

```diff
@@ -0,0 +1,31 @@
+import Image from "next/image";
+
+import { trpc } from "@/utils/trpc";
+import { getShortBaseUrl } from "@/utils/short-url";
+
+import Link from "@/components/content/Link";
+
+export function Schedule() {
+  const schedule = trpc.notifications.getSchedule.useQuery();
+
+  if (schedule.isLoading) return <p>Loading schedule...</p>;
+
+  if (!schedule.data) return <p>No schedule is available currently.</p>;
+
+  return (
+    <Link
+      href={`${getShortBaseUrl()}/l/${schedule.data.slug}`}
+      external
+      custom
+      className="group"
+    >
+      <Image
+        src={schedule.data.link}
```

Comment:
Should we do a very basic check if its an imgur image link?

Reply:
I thought about doing that, but decided I trust Mik enough to stick to using imgur, and if not, I kinda want to see it error so we know to fix it.